src/main/java/org/broadinstitute/hellbender/tools/copynumber/gcnv/GermlineCNVAbstractVariantComposer.java

```diff
@@ -0,0 +1,39 @@
+package org.broadinstitute.hellbender.tools.copynumber.gcnv;
+
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.writer.VariantContextWriter;
+import htsjdk.variant.vcf.VCFHeaderLine;
+import org.broadinstitute.hellbender.utils.Utils;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Base class for {@link GermlineCNVIntervalVariantComposer} and {@link GermlineCNVSegmentVariantComposer}.
+ *
+ * @author Mehrtash Babadi &lt;mehrtash@broadinstitute.org&gt;
+ */
+public abstract class GermlineCNVAbstractVariantComposer {
+    static final String VARIANT_PREFIX = "CNV";
+    static final Allele REF_ALLELE = Allele.create("N", true);
```

Comment:
We don't have much wiggle room here. The `Allele`-related code in GATK is pre-SV and is written with SNPs and INDELs in mind. An allele is considered _symbolic_ if it is like named like `<some_symbolic_description>`. A symbolic allele is meant to describe a complicated INDEL scenario, I believe, and can not be reference (exception is thrown otherwise). If the name doesn't start and end with `<` and `>`, respectively, then its name is assumed to represent alt bases and must only contain A/C/T/G/N (exception is thrown otherwise). So, we are left with valid bases for the REF allele. The natural choice is something non-informative, i.e.`N`.